client/fingerprint/consul.go

```diff
@@ -12,15 +12,21 @@ import (
 	"github.com/hashicorp/nomad/nomad/structs"
 )
 
+const (
+	consulAvailable   = "consulavailable"
+	consulUnavailable = "consulunavailable"
```

Comment:
Can we just change these to `available` and `unavailable`? That way you could just compare against the last state, and log a message saying something like `consul status changed to %s`.